### PR TITLE
Include branched callflow id + group id in CCVs

### PR DESCRIPTION
**Pull request by kalda341**
We would like more useful information to be stored in CDRs. I have added Group-ID and Branched-Callflow-ID to the Custom-Channel-Vars.
The accessors are used by existing code that I have not yet created pull requests for.

### CODE REVIEW

**Comment by lazedo** [comment]
@kalda341 thanks for this pull request. please include the other accessors at the time of your other pull request. also, `monster_group_id` is not an appropriate name, the api should be client agnostic.

**Comment by kalda341** [comment]
I agree that the name is poor, but I did not want it to be confused with the many other types of groups. Do you have any suggestions as to what to name it? Just `group_id`?

**Comment by kalda341** [comment]
@lazedo I have renamed monster_group_id to group_id.

File: applications/callflow/src/module/cf_callflow.erl [hunk 1]
```diff
@@ -31,5 +31,48 @@ handle(Data, Call) ->
...
+%%--------------------------------------------------------------------
+%% @private
+%% @doc
+%% Set the group_id field in the CCVs if the callflow is a baseGroup
+%% @end
+%%--------------------------------------------------------------------
+-spec maybe_add_group_id(kz_json:object(), kapps_call:call()) -> kapps_call:call().
+maybe_add_group_id(JObj, Call) ->
+    case kz_json:get_value(<<"type">>, JObj) of
+        <<"baseGroup">> ->
```
Comment (jamesaimonetti) [inline applications/callflow/src/module/cf_callflow.erl]: Where is "baseGroup" coming from? This seems very specific to your operation and not generally useful.

Comment (kalda341) [inline applications/callflow/src/module/cf_callflow.erl]: In Monster UI's VOIP app a baseGroup is the callflow that contains a ring group. We would like the group_id field to be included if the callflow element is a baseGroup.
You are correct - this is only useful for callflows created with Monster.

**Comment by kalda341** [comment]
@jamesaimonetti Any updates?

**Comment by lazedo** [comment]
@kalda341 this approach is very tied to a specific use case `baseGroup`. we need to figure out a different way of doing this without overloading CCVs.

**Comment by kalda341** [comment]
I agree. Maybe something like what you've done with custom SIP headers?

**Pull request closed without merging**